BP/common/decisions/BP_government_conversion_decisions.txt

```diff
@@ -49,3 +96,290 @@ convert_to_subroman_decision = {
 	ai_will_do = { base = 100 }
 }
 
+convert_to_bureaucratic_decision = {
+	picture = "gfx/interface/illustrations/decisions/decision_realm.dds"
+	major = yes
+	desc = convert_to_bureaucratic_decision_desc
+
+	ai_check_interval = 36
+
+	is_shown = {
+		OR = { has_government = subroman_government }
+		has_cultural_era_or_later = culture_era_late_migration
+	}
+
+	is_valid = {
+		custom_description = {
+			text = faith_is_reformed
+			subject = root
+			faith = { NOT = { has_doctrine_parameter = unreformed } }
+		}
+		is_independent_ruler = yes #Must be independent
+		highest_held_title_tier = 5 #Must be an empire
+		prestige_level >= 2
+		capital_province.county = { development_level >= convert_to_feudalism_development_requirement }
+		
+		custom_description = { #You need to have ALL (non-regional/unique) tribal innovations and roman heritage
+			text = has_all_innovations_for_subroman
+			subject = root.culture
+			culture = {
+				#All tribal innovations
+				has_innovation = innovation_vexilium
+				has_innovation = innovation_mustering_grounds
+				has_innovation = innovation_castrae
+				has_innovation = innovation_quilted_armor
+				has_innovation = innovation_development_00
+				has_innovation = innovation_leges_barbarorum
+				has_innovation = innovation_writing
+				has_innovation = innovation_barter
+				has_innovation = innovation_oppidum
+				#All Late Antiquity Innovations
+				has_innovation = innovation_standing_army
+				has_innovation = innovation_limitanei
+				has_innovation = innovation_catapult
+				has_innovation = innovation_mail_armor
+				has_innovation = innovation_muri
+				has_innovation = innovation_conscription
+				has_innovation = innovation_development_01
+				has_innovation = innovation_fora
+				has_innovation = innovation_coinage
+				has_innovation = innovation_mundium
+				has_innovation = innovation_state_religion
+				#Required Early Migration Innovations
+				has_innovation = innovation_migratory_tactics
+				has_innovation = innovation_migratory_defenses
+				has_innovation = innovation_double_entry_bookkeeping
+				has_innovation = innovation_tenant_farming
+				has_innovation = innovation_development_02
+				has_innovation = innovation_doctrine_and_dogma
+				#The late migration era innovation
+				has_innovation = innovation_civil_service_revitalization
+			}
+		}
+	}
+
+	cost = { prestige = 250 }
+
+	effect = {
+		change_government = bureaucratic_government
+		add_realm_law_skip_effects = bureaucratic_authority_2
+		add_realm_law_skip_effects = imperial_elective_succession_law
+	}
+
+	ai_potential = { always = yes }
+
+	ai_will_do = { base = 100 }
+}
+
+convert_to_prefeudal_decision = {
```

Comment:
Pre-Feudal is meant to be a dead end, you don't convert into it. If you are High Tribal the aim is to become Feudal.